scripts/controlnet.py

```diff
@@ -43,53 +44,6 @@
 except ImportError:
     pass
 
-# svgsupports
-svgsupport = False
-try:
-    import io
-    from svglib.svglib import svg2rlg
-    from reportlab.graphics import renderPM
-    svgsupport = True
-except ImportError:
-    pass
-
-# Note: Change symbol hints mapping in `javascript/hints.js` when you change the symbol values.
-refresh_symbol = '\U0001f504'       # 🔄
-switch_values_symbol = '\U000021C5' # ⇅
-camera_symbol = '\U0001F4F7'        # 📷
-reverse_symbol = '\U000021C4'       # ⇄
-tossup_symbol = '\u2934'
-trigger_symbol = '\U0001F4A5'  # 💥
-open_symbol = '\U0001F4DD'  # 📝
-
-webcam_enabled = False
```

Comment:
Moved to class member variable of ui.ControlNetUnit. Do we really want this toggle to be global instead of per ControlNetUnit?

Reply:
I don't think it should have been global in the first place. However, does it work to track multiple webcams at the same time? I cannot test it because I do not own one.

Reply:
I tested that when multiple controlnet units are enabled. Their states do get messed up. We should definitely change the state to per-controlnet unit.